api/rosetta/balance.go

```diff
@@ -15,12 +15,12 @@
 package rosetta
 
 import (
-	"errors"
+	errortype "errors"
```

Comment:
Since it is used for type assertions (`errors.As`) and not really for its type, I'm not fond of the alias naming here 🤔 I think I would have just kept this one unaliased and aliased rosetta/errors to `rerr` or something like that. WDYT?

Reply:
Yeah, I'm not too fond of the alias either. Initially I had the rosetta/errors aliased to `rosettaErrors`. 

I disliked that because
a) it's just an ugly name
b) really in the code the accent is more on the rosetta errors. I don't like the fact that the standard lib `errors` package takes the nice name even though it's only referenced in a line or two.

I think I'd prefer thinking of a nicer alias :/ Something like `stderr` or `stderror` comes to mind and is nice but that's really the output stream (like `stdout`/`stderr`).

I've also thought of just wrapping the `errors.As` in the `rosetta/errors` package but that feels dirty/hackish.

Reply:
Yeah I thought the same about wrapping `errors.As` haha. I think I prefer the original alias of `rosettaErrors` then, or even `rosettaErr` if we want to keep it relatively short.

Reply:
I think the `rosetta/errors` package should be renamed. That's the exact reason why I used `failure` earlier. You could use `fail` as well, that reads best imo. I don't know why you changed it, though? `errors.WithError` doesn't read well at all, for instance.